packages/data-stores/src/site/constants.ts

```diff
@@ -1,3 +1,7 @@
+import { isEnabled } from '@automattic/calypso-config';
+
 export const STORE_KEY = 'automattic/site';
 
-export const PLACEHOLDER_SITE_ID = 224076220; // creatio2demo.wordpress.com
+export const PLACEHOLDER_SITE_ID = isEnabled( 'pattern-assembler/v2' )
```

Comment:
This may not work if the file is loaded before the calypso-config is initialized

Reply:
I'm understanding that you are know that case will happen and we have to find another way to use a different constant. Is there another way to do that?

Reply:
Another way is to turn the constant into function, like: https://github.com/Automattic/wp-calypso/pull/84960

Reply:
Thanks 😊